tool_path_planner/include/tool_path_planner/plane_slicer_raster_generator.h

```diff
@@ -22,22 +22,16 @@
 #ifndef INCLUDE_PLANE_SLICER_RASTER_GENERATOR_H_
 #define INCLUDE_PLANE_SLICER_RASTER_GENERATOR_H_
 
-#include <vtkPoints.h>
-#include <vtkSmartPointer.h>
-#include <vtkPolyData.h>
+#include <tool_path_planner/path_generator.h>
+
 #include <boost/optional.hpp>
-#include <pcl/PolygonMesh.h>
-#include <shape_msgs/Mesh.h>
-#include <vtkCellLocator.h>
-#include <vtkKdTreePointLocator.h>
 #include <jsoncpp/json/json.h>
-#include <ros/console.h>
-
-#include <tool_path_planner/path_generator.h>
+#include <pcl/PolygonMesh.h>
 
 namespace tool_path_planner
 {
-class PlaneSlicerRasterGenerator : public PathGenerator
+class [[deprecated("Replaced by PlaneSliceRasterPlanner in noether_tpp")]] PlaneSlicerRasterGenerator
```

Comment:
Oh my gosh.  Deprecation warnings telling you what new code to use?

IT'S HAPPENING!!!